test/src/org/exist/security/internal/BackupRestoreSecurityPrincipalsTest.java

```diff
@@ -0,0 +1,291 @@
+/*
```

Comment:
that was backup test that also check security related staff. why did you move it out of backup related tests package?

Reply:
Good question! Again, I don't know - as it was a change that you made in your original PR - https://github.com/eXist-db/exist/pull/1056/files#diff-14915bb0f81a3900e16d841b355f91b9

Although after some consideration, this does now feel like the correct place for it after some.

Reply:
right. guess, I had exactly same reasoning that time too